examples/algorithms/plot_junction_tree_algorithm.py

```diff
@@ -0,0 +1,19 @@
+import networkx as nx
```

Comment:
The examples need a docstring at the top of the file.  This is required by https://sphinx-gallery.github.io/stable/index.html, which we use to make the gallery from the examples.

See, for example, `https://github.com/networkx/networkx/blob/master/examples/algorithms/plot_beam_search.py`, which starts:
```
"""
===========
Beam Search
===========

Beam search with dynamic beam width.

The progressive widening beam search repeatedly executes a beam search
with increasing beam width until the target node is found.
"""
```

Reply:
This should fix the Travis CI error.